### ISSUE TITLE
Unable to generate documentation

### ISSUE THREAD

**Issue by FFY00** [pyudev/pyudev#366]
```
$ sphinx-build -b html -d doc/doctrees doc doc/html
Running Sphinx v2.0.1
making output directory... done

Extension error:
Unknown event name: b'autodoc-process-docstring'
```

If I remove the `b` it works, is that expected?

**Comment by mulkieran** [comment]
The b looks unintentional so I'm removing it.

**Comment by mulkieran** [comment]
Closed by #390
